src/main/java/ch/uzh/ifi/hase/soprafs22/user/RegisteredUser.java

```diff
@@ -0,0 +1,65 @@
+package ch.uzh.ifi.hase.soprafs22.user;
+
+public class RegisteredUser extends UserDecorator {
```

Comment:
We would need to add the JPA annotations here, right? (like @Column etc.) Would this work?

Reply:
I think we can use this, where you embed the user 
https://www.baeldung.com/jpa-embedded-embeddable

Reply:
thx!
added the annotations.